terraform/main.tf

```diff
@@ -0,0 +1,52 @@
+provider "google" {
+  project = "home-server-347301"
```

Comment:
I'm unfamiliar with Terraform's template language so I'll take your word for it that this works. 😊 But the project name here seems oddly specific.

Reply:
Nice catch, this should have come from the tfvars too